.github/workflows/rspec-test.yml

```diff
@@ -67,10 +67,20 @@ jobs:
               npm run test
           fi
 
-      - name: Test
-        if: ${{ steps.secret-check.outputs.available != 'true' }}
+      - name: Run RSpec to Output File
+        id: rspec_test
         run: |
-          bundle exec rspec
+          bundle exec rspec --format json --out ${{ github.workspace }}/rspec_output.json
+          
+      - name: Checkout public-reusable
+        uses: actions/checkout@v4
+        with:
+          repository: 'Strongmind/public-reusable-workflows'
```

Comment:
Pulling down the whole repo just to run a script seems...smelly. Isn't there a way to download a single file from a public github repo?